docs/reference/worldinfo.md

```diff
@@ -5,7 +5,7 @@ WorldInfo {
   SFString title                          ""         # any string
   MFString info                           [ ]        # any string
   SFFloat  gravity                        9.81       # [0, inf)
-  SFFloat  CFM                            0.00001    # [0, inf)
+  SFFloat  CFM                            0.00001    # ]0, inf)
```

Comment:
I am not familiar with this notation, shouldn't it be `(` instead of `]`?

Reply:
Yes, you are perfectly right:
- https://socratic.org/questions/how-do-you-know-when-to-use-brackets-or-parenthesis-in-finding-domain-or-range
- https://hannahtaylormath.weebly.com/domain-and-range.html

I used the closing bracket in my math courses, but that is maybe something not very standard. I will change it.